lib/Mojo/Message.pm

```diff
@@ -97,10 +97,14 @@ sub fix_headers {
   # Content-Length or Connection (unless chunked transfer encoding is used)
   my $content = $self->content;
   my $headers = $content->headers;
+  my $websocket = $headers->sec_websocket_accept;
+  my $is_websocket = defined $websocket;
+  my $size = $self->body_size;  
```

Comment:
You're calling `body_size` twice now.

Reply:
fixed

Reply:
And calling it here does have a performance cost for some cases.

Reply:
Does 62138b8 work? Sorry I never did any perl. This is just trying to help a vendor to connect to our backend

Reply:
ok fixed that